core.js

```diff
@@ -0,0 +1 @@
+export { default } from './src/core.js'
```

Comment:
instead of an alias to `./src/core.js`, this should be a full copy of that file

likewise, *every* file in `src` should be copied outside of `src`, and then exposed for distribution